contracts/ConvergentCurvePool.sol

```diff
@@ -449,7 +482,7 @@ contract ConvergentCurvePool is IMinimalSwapInfoPool, BalancerPoolToken {
                     amountIn.sub(amountOut)
                 );
                 // we record that collected fee from the input bond
-                feesBond += uint128(impliedYieldFee);
+                feesBond += uint120(impliedYieldFee);
```

Comment:
This change needs to be reverted, because the type of `feesBond` has been restored to `uint128`.